packages/sequelize/src/adapter/index.js

```diff
@@ -0,0 +1,416 @@
+import "core-js/stable";
+import "regenerator-runtime/runtime";
+import { createHash, randomBytes } from "crypto";
+
+import { CreateUserError } from "./lib/errors";
+import logger from "./lib/logger";
```

Comment:
```suggestion
```

You will find this in `appOptions` in `getAdapter`. You can remove `./lib/logger`